snuba/admin/static/runtime_config/index.tsx

```diff
@@ -44,10 +61,17 @@ function RuntimeConfig(props: { api: Client }) {
     });
   }
 
+  function fetchDescriptions() {
```

Comment:
Can you explain why you need this? I'm not sure why you would fetch descriptions without fetching the associated config value. Is this for "deleted but still described" configs?

Reply:
Yep, this would be called in the "new config" flow.

When we add a new config:
- We fetch all available descriptions (which includes descriptions saved but config was deleted)
- https://github.com/getsentry/snuba/blob/b44edbde95b5f9aaedf26ec5a3fa52599103d264/snuba/admin/static/runtime_config/index.tsx#L70-L75
- As we write the name for the config, we check the map of descriptions to see whether we already have a description for this key
- If it's found, the description box is populated
- https://github.com/getsentry/snuba/blob/b44edbde95b5f9aaedf26ec5a3fa52599103d264/snuba/admin/static/runtime_config/index.tsx#L185-L192